src/coreclr/vm/ceeload.cpp

```diff
@@ -4589,7 +4589,7 @@ Module::GetAssemblyIfLoaded(
                 DomainAssembly * pCurAssemblyInExamineDomain = GetAssembly()->GetDomainAssembly();
                 if (pCurAssemblyInExamineDomain == NULL)
                 {
-                    continue;
```

Comment:
Some of the preceding lines indicate that this used to be a useful loop.  That being the case, this should probably either stand (if coreclr is every anticipated to get more than one app domain), or things should possibly be reworked to remove the loop entirely (this is not necessarily as simple as it first appears, due to the resulting `goto`-like behavior).